inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.html

```diff
@@ -110,33 +138,144 @@ <h1>
                       </button>
                     </div>
                   </form>
-                </div>
+                </span>
               </span>
               <span class="dropdown" aria-haspopup="true" aria-expanded="false">
-                <button class="btn btn-action btn-secondary dropdown-toggle flex-content" type="button" data-toggle="dropdown">
-                  <i class="fas fa-cog"></i>&nbsp;<wicket:message key="settings" />
+                <button class="btn btn-action btn-secondary dropdown-toggle flex-content dd-toggle"
+                  type="button">
+                  <i class="fas fa-user"></i>&nbsp;
+                  <wicket:message key="Annotators"/>
                 </button>
-                <div class="dropdown-menu shadow-lg pt-0 pb-0" role="menu" style="min-width: 800px;">
-                  <div class="card-header">
-                    <wicket:message key="settings" />
+                  <script>
+                      let dropDownMenuButton = $('button.dd-toggle');
+
+                      dropDownMenuButton.on('click', function () {
+                          $('form.ddform').show();
+                      });
+
+                      $('body').on('click', function (e) {
+                          if (!dropDownMenuButton.is(e.target)
```

Comment:
Is this really necessary? Please at least comment what you are doing here and why this is necessary.